vendor/github.com/hashicorp/go-cleanhttp/cleanhttp.go

```diff
@@ -32,6 +32,7 @@ func DefaultPooledTransport() *http.Transport {
 		IdleConnTimeout:       90 * time.Second,
 		TLSHandshakeTimeout:   10 * time.Second,
 		ExpectContinueTimeout: 1 * time.Second,
+		ForceAttemptHTTP2:     true,
```

Comment:
Is switch to http2 worth calling out in changelog?

Reply:
Good call, will do.